sqlengine/postgres_engine.go

```diff
@@ -1,43 +1,55 @@
 package sqlengine
 
 import (
+	"bytes"
 	"database/sql"
 	"fmt"
+	"text/template"
 
 	"github.com/lib/pq" // PostgreSQL Driver
 
 	"code.cloudfoundry.org/lager"
 )
 
-const defaultStateDBName = "broker_state"
+var ensureTriggerPattern = `
```

Comment:
The advantage of doing this in the global scope is that the templates only get compiled once at startup instead of being compiled every time.